### ISSUE TITLE
[enhancement] disable scanning

### ISSUE THREAD

**Issue by eur0pa** [michenriksen/aquatone#124]
Hello, and thanks for the new version.
It'd be great having a switch allowing aquatone to just screenshot a set of given URLs without performing any portscan on them.
Thanks!

**Comment by eur0pa** [comment]
@michenriksen my bad. Second issue: SSL certificates are being verified; hosts with a bad certificate will result in a blank screenshot.

**Comment by michenriksen** [comment]
SSL verification is turned off and it works fine for me. Chrome is a bit unreliable in headless mode, which is why the tool recommends installing Chromium. Are you using that?

**Comment by eur0pa** [comment]
Yup, using Chromium, manually specifying its path. Blank capture. Repro:

```
echo 'https://expired.badssl.com' | aquatone -chrome-path ~/Applications/Chromium.app/Contents/MacOS/Chromium-orig -out test
```

**Comment by eur0pa** [comment]
Is this getting worked on?
